frontend/content/users/alex-pulver/index.html

```diff
@@ -0,0 +1,4 @@
+name: Alex Pulver
```

Comment:
The file should have a `.yml` extension.

Reply:
Oops, sorry! Will fix